scout/utils/ensembl_rest_clients.py

```diff
@@ -76,19 +78,22 @@ def send_request(self, url):
         """
         data = {}
         try:
-            request = Request(url, headers=HEADERS)
-            response = urlopen(request)
-            content = response.read()
-            if content:
-                data = json.loads(content)
-        except HTTPError as e:
-            LOG.info("Request failed for url {0}: Error: {1}\n".format(url, e))
-            data = e
-        except ValueError as e:
-            LOG.info("Request failed for url {0}: Error: {1}\n".format(url, e))
-            data = e
+            response = requests.get(url, headers=HEADERS)
```

Comment:
Have you tested that this works al the old code?

Reply:
Yes